ios/brave-ios/Sources/BraveWallet/Crypto/WalletActions/SwapCryptoView.swift

```diff
@@ -241,8 +241,8 @@ struct SwapCryptoView: View {
 
   /// The DEX Aggregator for the current network.
   var dexAggregator: DEXAggregator {
-    // TODO(stephenheaps): Once LiFiQuote is supported, we need to
-    // update this disclaimer to include LiFi description & privacy
+    // TODO(stephenheaps): We may need to remove and/or update
+    // this disclaimer to include LiFi description & privacy
```

Comment:
maybe a link to the slack thread discussing the update of the disclaimer copy.

Reply:
We're waiting on some info for this change: https://bravesoftware.slack.com/archives/C023VS4HJ6Q/p1711451893091189?thread_ts=1708974083.632369&cid=C023VS4HJ6Q

FWIW LiFi is not yet enabled in this PR.